app/components/distributor_title_component/distributor_title_component.html.haml

```diff
@@ -0,0 +1 @@
+%h3 #{@name}
```

Comment:
I've never seen this syntax. Thanks for pointing me to it. This example is slightly simpler like this:
```suggestion
%h3= @name
```
The `=` syntax is also a bit more robust for Ruby code.

> Haml uses an overly simplistic regular expression to identify string interpolation rather than a full-blown Ruby parser. This is fast and works for most code but you may have errors...
> https://haml.info/docs/yardoc/file.REFERENCE.html#gotchas